### PR TITLE
14 feature request using local filesystem by default

### LINKED ISSUE #14: [Feature request] Using local filesystem by default
Hi,

I was going to do all this setup by hand, and found this this (gratefully!)
I was thinking that maybe the default behaviour for a localstack deployment should be using  --local https://www.pulumi.com/docs/concepts/state/#local-filesystem or a file://<dir> backend

This would make more sense as a lot of people will not persist their localstack environments (for example, i use it in a docker-compose service)

Cheers

### PR DESCRIPTION
**Pull request by lakkeger**
# Motivation
README contains parts in example which not clear enough. 

# Changes
- [x] remove original aws package performance disclaimer
- [x] separate and add explanation on local backend setup
- [x] add notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,15 +23,25 @@ The `pulumilocal` command has the same usage as the `pulumi` command. For detail
 please refer to the man pages of `pulumi -h`.
 
 For example:
```
Comment (whummer) [inline README.md]: nit: I guess we could remove this line

File: README.md [hunk 2]
```diff
@@ -24,15 +23,25 @@ The `pulumilocal` command has the same usage as the `pulumi` command. For detail
...
+
+### Add environment variables to store state on local backend (optional)
 ```shell
-mkdir myproj
 export PULUMI_CONFIG_PASSPHRASE=lsdevtest
 export PULUMI_BACKEND_URL=file://`pwd`/myproj
+```
+_Note: For further options please consult the official documentation on available [environment variables][env_vars] and [local backend][local_backend]._
+
+[env_vars]: https://www.pulumi.com/docs/cli/environment-variables/
```
Comment (whummer) [inline README.md:32]: Wow, wasn't aware of this markdown syntax to reference links! (usually always used the format `[link text](http://...)`). Nice one! :)

File: README.md [hunk 3]
```diff
@@ -24,15 +23,25 @@ The `pulumilocal` command has the same usage as the `pulumi` command. For detail
...
+
+### Create a new Pulumi Project with stack name lsdev
+```shell
+mkdir myproj
 pulumilocal new typescript -y -s lsdev --cwd myproj
 ```
+_Note: `--cwd` switch is unnecessary if commands are being run in project directory._
 
-### Select and Create the lsdev Pulumi Stack
+### Select and Create the lsdev Pulumi stack
```
Comment (whummer) [inline README.md]: small nit: if we lowercase `stack`, then we could also change the casing of `create`:
```suggestion
### Select and create the lsdev Pulumi stack
```

**Review by whummer — APPROVED** [review]
Nice changes, kudos for making the README a bit more clear @lakkeger ! 👍

File: README.md (hunk 2 above)
Comment (lakkeger) [reply README.md:32]: Glad you like it, thought it provides better markdown readability than dropping it inline :)

**Pull request merged**